client/src/components/ProfileEdit.jsx

```diff
@@ -0,0 +1,182 @@
+import React, { Component } from 'react';
+import { Row, Col, Input, Button } from 'react-materialize'
+import { request } from '../utils'
+import { Link } from 'react-router-dom'
+
+
+class ProfileEdit extends Component {
+
+    state = {
+        user_edit: {},
+        error_nick: "",
+        error_repeated: "",
+        error_first: "",
+        error_last: "",
+        error_trigger_nick: false,
+        error_trigger_first: false,
+        error_trigger_last: false
```

Comment:
Why do you need separate variable for each field, why don't use array or mapping instead?